documentation/jetty/modules/code/examples/src/main/java/org/eclipse/jetty/docs/programming/server/ServerDocs.java

```diff
@@ -176,7 +176,7 @@ public void onOpen()
         @Override
         public void onFillable()
         {
-            callback.iterate();
+            callback.succeeded();
```

Comment:
This should be as before, `callback.iterate()`, because `onFillable()` is called by the `fillInterested()` call from `onOpen()`.

It's inside `process()` that we do things differently.

Reply:
Ah nice way to solve the difference between the  first "demand" and the subsequent demand.